server/lib/actors/fetcher.go

```diff
@@ -0,0 +1,81 @@
+package actors
+
+import (
+	"fmt"
+	"database/sql"
+	"time"
+
+	"github.com/umegaya/channer/server/lib/models"
+	proto "github.com/umegaya/channer/server/proto"
+
+	"github.com/umegaya/yue"
+)
+
+var FLAME_THRESHOLD float64 = 0.1
+var FLAME_VOTE_THRESHOLD uint32 = 10
+
+func topicFetcher(btyp proto.TopicListRequest_BucketType, start, end time.Time, locale string) ([]FetchResult, error) {
+	entries := make([]FetchResult, DEFAULT_HOTBUCKET_SIZE)
+	dbm := models.DBM()
+	var rows *sql.Rows
+	var err error
+	locale_clause := ""
+	if len(locale) > 0 {
+		locale_clause = fmt.Sprintf(" and locale = '%s'", locale)
+	}
+	switch btyp {
+	case proto.TopicListRequest_Rising:
+		where_clause := fmt.Sprintf("created >= %v and created < %v%s", yue.UUIDAt(start), yue.UUIDAt(end), locale_clause)
+		rows, err = dbm.Query(
```

Comment:
__[gometalinter]__
* [aligncheck] invalid operation: dbm (variable of type *github.com/umegaya/channer/server/lib/models.Database) has no field or method Query